.gitmodules

```diff
@@ -2,3 +2,7 @@
 	path = lib/node-replication
 	url = git@github.com:vmware/node-replication.git
 	branch = mut_scan_vec
+[submodule "nrk-dcm-scheduler"]
```

Comment:
@gz Now that I can use github releases, is there a reason to leave this as a submodule? I couldn't think of one, so if you can't either, I will remove it as a submodule.

Reply:
Yes lets drop the submodule again, but we should have some way to override the jar with a local custom version (maybe this can be part of run.py though)...
Maybe run.py can also contain the build commanda and we just point it to the right directory in this case?

Reply:
just fyi removing a submodule again
is quite the hassle but these steps tend to work: https://nrkernel.systems/book/development/Configuration.html#removing-a-submodule-in-the-repository

Reply:
Thank you, I should have looked it up first. Apologies!

Reply:
Is there anything else I should do to fix things further on the runners, since I didn't remove the submodule gracefully?